packages/resolve-scripts/src/configs/webpack.client.config.js

```diff
@@ -37,6 +37,7 @@ module.exports = {
 
     plugins: [
         new webpack.DefinePlugin({
+            'process.env': JSON.stringify(process.env),
```

Comment:
I got it, but I don't understand, why it is needed to define both, if NODE_ENV placed inside proces.env.